kloppy/_providers/wyscout.py

```diff
@@ -14,6 +16,7 @@ def load(
     event_types: Optional[List[str]] = None,
     coordinates: Optional[str] = None,
     event_factory: Optional[EventFactory] = None,
+    data_model_version: Optional[str] = None,
```

Comment:
Did you include the term `model` in the variable name on purpose? I feel that it doesn't add much and can perhaps even be confusing in the context of football analytics, where the term _model_ is often used for different purposes. How about `data_version` or `data_specification`?